LaunchSpatial.bat

```diff
@@ -8,6 +8,6 @@ spatial worker build build-config
 rem To launch a local Spatial deployment with a different launch configuration, change the default_launch.json argument to the name of your configuration file, 
 rem for example one_worker_test.json or two_worker_test.json. Similarly, you can specify a different snapshot than the default one by replacing the value of 
 rem the --snapshot argument with the path to the snapshot you want to launch with.
-spatial local launch default_launch.json --snapshot=snapshots/default.snapshot --runtime_version=15.0.1
+spatial local launch default_launch.json --snapshot=snapshots/default.snapshot --runtime_version=15.3.0
```

Comment:
Why is this 15.3.0? Shouldn't it be 15.1.1?

Reply:
doh

Reply:
```suggestion
spatial local launch default_launch.json --snapshot=snapshots/default.snapshot --runtime_version=15.1.1
```